ambry-server/src/main/java/com.github.ambry.server/AmbryServer.java

```diff
@@ -155,11 +157,13 @@ public void startup() throws InstantiationException {
         ports.add(new Port(nodeId.getSSLPort(), PortType.SSL));
       }
 
+      storeKeyConverterFactory = Utils.getObj(serverConfig.serverStoreKeyConverterFactoryClass, properties, registry);
+
       networkServer = new SocketServer(networkConfig, sslConfig, registry, ports);
       requests =
           new AmbryRequests(storageManager, networkServer.getRequestResponseChannel(), clusterMap, nodeId, registry,
               findTokenFactory, notificationSystem, replicationManager, storeKeyFactory,
-              serverConfig.serverEnableStoreDataPrefetch);
+              serverConfig.serverEnableStoreDataPrefetch, storeKeyConverterFactory);
```

Comment:
`ReplicationManager` needs the `storeKeyConverterFactory` too right? any other components?

Reply:
Added.  I don't believe any others.